Formula/gst-plugins-good.rb

```diff
@@ -39,7 +39,7 @@ class GstPluginsGood < Formula
   depends_on "gdk-pixbuf" => :optional
   depends_on "aalib" => :optional
   depends_on "cairo" => :optional
-  depends_on "flac" => [:optional, "with-libogg"]
+  depends_on "flac" => :optional
```

Comment:
despite the audit error we'll need to keep this the same until a proper solution is implemented.